### DISCUSSION TITLE
Documentation for Job Recovery Process

### DISCUSSION THREAD

**Discussion by mkstephenson** [quartznet/quartznet#2298] [Q&A]
Hi, I wasn't able to find anything in the docs (or I am blind and unable to use search properly) but I was wondering if the job recovery process is documented somewhere and what quartz does when a job goes into recovery?

**Comment by lahma** [comment]
I'm afraid that it is poorly documented. I would advise lookin at the source of Quartz and especially `MisfireHandler`.
